### ISSUE TITLE
simulator sound blocked on mac safari until restart or mute is pressed

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#3259]
**Describe the bug**

Mac safari's up to date with iOS safari and is now blocking auto play sounds; we should probably start muted on mac safari (like we do on ios) to account for this

**To Reproduce**
1. Go to a new project on mac safari
2. Add a block that makes sound
3. See that there is no sound until you restart the simulator

**Expected behavior**

start muted so it's clear that sound is off

**Desktop (please complete the following information):**
 - OS: mac
 - Browser  safari
 - Version 13.0.3 (15608.3.10.1.4)

**Comment by abchatra** [comment]
Won't fix safari sound issues.
